telebot/handler_backends.py

```diff
@@ -266,5 +261,10 @@ class CancelUpdate:
     of post_process in middlewares.
     """
 
-    def __init__(self) -> None:
-        pass
\ No newline at end of file
+
+class Continue:
```

Comment:
Renaming this to ContinueHandling would be more appropriate.. What do you think?